src/ProjectNeon/Assets/Scripts/Saver.cs

```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+
+
+[CreateAssetMenu(fileName = "New Saver", menuName = "Saver")]
+public class Saver : ScriptableObject
+{
+    // @todo #281: 30min Add metod to check if the one key is used more then ones
+    public static void Save(KeysEnum saveKey, string data)
```

Comment:
These methods should be private. We only need public methods for gettings and setting SoundVolume and MusicVolume.

Having all of these public methods will lead to error if people don't know which data types are used.

Reply:
Wouldn't it be necessary to save a player’s progress if he leaves the game in the middle of the session?